config.yml

```diff
@@ -779,6 +945,265 @@ production_factories:
     repair_inputs:
       Diamond:
         material: DIAMOND
+  Random_Wooden_Tool_Factory:
+    name: Random Wooden Tool Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Sticks:
+        material: STICK
+        amount: 96   
+    recipes:
+      - Create_Wooden_Axe
+      - Create_Wooden_Spade
+      - Create_Wooden_Axe
+      - Create_Fishing_Rod
+    repair_multiple: 2
+    repair_inputs:
+      Sticks:
+        material: STICK
+        amount: 9  
+  Random_Leather_Armor_Factory:
+    name: Random Leather Armor Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Leather:
+        material: LEATHER
+        amount: 64   
+    recipes:
+      - Create_Leather_Leggings
+      - Create_Leather_Boots
+      - Create_Leather_Chestplate
+      - Create_Leather_Helmet
+    repair_multiple: 2
+    repair_inputs:
+      Sticks:
+        material: STICK
+        amount: 6  
+  Random_Wooden_Weapon_Factory:
+    name: Random Wooden Weapon Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Sticks:
+        material: STICK
+        amount: 32  
+    recipes:
+      - Create_Wooden_Sword
+    repair_multiple: 2
+    repair_inputs:
+      Sticks:
+        material: STICK
+        amount: 3  
+  Random_Iron_Tool_Factory:
+    name: Random Iron Tool Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 96   
+    recipes:
+      - Create_Iron_Axe
+      - Create_Iron_Pickaxe
+      - Create_Iron_Spade

```

Comment:
Hoes are excluded from random tool factory?

I'm guessing shears are intentionally left in the Iron Forge, yes?